package.json

```diff
@@ -10,6 +10,9 @@
     "@types/node": "^16.11.38",
     "@types/react": "^18.0.10",
     "@types/react-dom": "^18.0.5",
```

Comment:
"@types"  need to be in devDependencies because in production we don't use typescript